src/api.raml

```diff
@@ -441,6 +441,28 @@ traits:
                                 15278:
                                     displayName: '15278'
                                     value: !include samples/article-vor/v2/15278.json
+                        application/vnd.elife.article-vor+json;version=3:
+                            (experimental):
+                            schema: !include model/article-vor.v3.json
+                            examples:
+                                minimum:
+                                    displayName: Minimum
+                                    value: !include samples/article-vor/v3/minimum.json
+                                complete:
+                                    displayName: Complete
+                                    value: !include samples/article-vor/v3/complete.json
+                                unpublishedMinimum:
+                                    displayName: PreviewMinimum
+                                    value: !include samples/article-vor/v3/unpublished-minimum.json
+                                unpublishedComplete:
+                                    displayName: PreviewComplete
+                                    value: !include samples/article-vor/v3/unpublished-complete.json
+                                00569:
+                                    displayName: '00569'
+                                    value: !include samples/article-vor/v3/00569.json
+                                15278:
+                                    displayName: '15278'
```

Comment:
are these two articles (569, 15278) special in some way or just plain examples of actual articles?

Reply:
Want to say it was the latter. 🤔